techniques/css/C37.html

```diff
@@ -2,86 +2,66 @@
+      <h1>画像を適応させるために、CSS の max-width と height を使用する</h1>
       <section id="important-information">
-         <h2>Important Information about Techniques</h2>
-         <p>See <a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">Understanding Techniques for WCAG Success Criteria</a> for important information about the usage of these informative techniques and how
-            they relate to the normative WCAG 2.1 success criteria. The Applicability section
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
-         </p>
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。</p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>This technique is applicable to Cascading Style Sheet / HTML technologies.</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/reflow">Success Criterion 1.4.10: Reflow</a> (Advisory)</span>.
+            <h2>適用 (対象)</h2>
+            <p>この達成方法は、カスケーディングスタイルシート / HTML 技術に適用される。</p>
+            <p>これは<span><a href="https://waic.jp/docs/WCAG21/Understanding/reflow">達成基準 1.4.10: リフロー</a> (十分な達成方法) </span>に関連する達成方法である。</p>
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to be able to present images without introducing
-               a horizontal scroll bar at a width equivalent to 320 CSS pixels, or a vertical scroll
-               bar at a height equivalent to 256 CSS pixels for content intended to scroll horizontally.
-               This is done by using CSS <code>max-width</code> and <code>height</code> property techniques that adapt to the available space and keep the original dimensions
-               of the image.
-            </p>
-            <p>Responsive layouts can add or remove columns or layout blocks, and each part of the
-               layout can be wider or smaller at different points. This technique ensures images
-               do not break out of their layout area, including in one-column layouts where it would
-               cause scrolling.
-            </p>
-            <p>The basic principles of fitting images are to:</p>
+            <h2>解説</h2>
+            <p>この達成方法の目的は、320 CSS ピクセルに相当する幅において横スクロールバーを、又は 256 CSS ピクセルに相当する高さにおいて縦スクロールバーを導入することなく、画像を表示できるようにすることである。これは、画像の元のサイズを維持しつつ使用可能なスペースに適応させる、CSS の <code>max-width</code> 及び <code>height</code> プロパティ手法を用いることによって行われる。</p>
```

Comment:
1文目の文末の for content intended to scroll horizontally. がまるっと抜けてしまっているようです。

```suggestion
            <p>この達成方法の目的は、横方向にスクロールすることを目的としたコンテンツの場合、320 CSS ピクセルに相当する幅において横スクロールバーを、又は 256 CSS ピクセルに相当する高さにおいて縦スクロールバーを導入することなく、画像を表示できるようにすることである。これは、画像の元のサイズを維持しつつ使用可能なスペースに収める、CSS の <code>max-width</code> 及び <code>height</code> プロパティ手法を用いることによって行われる。</p>
```